test/integration/targets/vmware_guest_instantclone/tasks/main.yml

```diff
@@ -0,0 +1,9 @@
+# Test0001: Create an instant clone
+- name: 0001 - Create an instant clone
+    validate_certs: False
+    hostname: '{{ vcenter_hostname }}'
+    username: '{{ vcenter_username }}'
+    password: '{{ vcenter_password }}'
+    datacenter: '{{ dc1 }}'
+    name: '{{ source_vm }}'
+    clone_name: '{{ name }}'
```

Comment:
You need to initialize the environment with `prepare_vmware_test`. I used this playbook to test your module:
```yaml
- import_role:
    name: prepare_vmware_tests
  vars:
    setup_attach_host: true
    setup_datastore: true
    setup_virtualmachines: true

- name: set state to poweron the first VM
  vmware_guest_powerstate:
    validate_certs: no
    hostname: "{{ vcenter_hostname }}"
    username: "{{ vcenter_username }}"
    password: "{{ vcenter_password }}"
    name: "{{ virtual_machines[0].name }}"
    folder: '{{ f0 }}'
    state: powered-on

# Test0001: Create an instant clone
- name: 0001 - Create an instant clone
  vmware_guest_instantclone:
    validate_certs: False
    hostname: '{{ vcenter_hostname }}'
    username: '{{ vcenter_username }}'
    password: '{{ vcenter_password }}'
    datacenter: '{{ dc1 }}'
    name: "{{ virtual_machines[0].name }}"
    clone_name: test_vm1
```